.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+### Description
+
+[Please describe what this change achieves]
+
+### Definition of Done
+
+### Issues Resolved
+
+[List any existing issues this PR resolves, or any Discourse or
+StackOverflow discussion that's relevant]
+
+### Check List
+
+- [ ] Spell Check
+- [ ] Docs actually built?
+- [ ] All tests pass
+- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://docs.chef.io/community_contributions.html#developer-certification-of-origin-dco>

```

Comment:
DCO signoff is not required for the docs IIRC.

Reply:
According to that doc, it is. :-)

Reply:
All docs repos were exempted in https://github.com/chef/chef-rfc/blob/master/rfc080-dco.md#downstream-impact and dcob doesn't run here so I suspect the docs need updating. We need a `:yodawg:` :)

Reply:
:yodawg: I'll take it out of that doc. TTYL!